db/migrate/20190611104619_add_index_to_audit_type.rb

```diff
@@ -0,0 +1,9 @@
+class AddIndexToAuditType < ActiveRecord::Migration[5.2]
+  def up
+    add_index :audits, :auditable_type
+  end
+
+  def down
+    remove_index :audits, :auditable_type
+  end
+end
```

Comment:
We already have this index in:

    t.index ["auditable_type", "auditable_id", "version"], name: "index_audits_on_auditable_type_and_auditable_id_and_version"

This was recently reviewed and removed by me and @tbrisker

Reply:
FYI recently there's been reported a performance related bug for audits and this fixed it: https://github.com/theforeman/foreman/pull/6749/files maybe it's related?